src/defmt.rs

```diff
@@ -244,6 +246,15 @@ pub fn decode_rtt<W: Write>(
         }
 
         for (secondary_index, core) in secondary_cores.iter_mut().enumerate() {
+            log::debug!("Secondary core index: {}, active cores: {}", secondary_index, active_cores);
```

Comment:
```suggestion
            log::debug!("Secondary core index: {}, active cores: {}", secondary_index + 1, active_cores);
```

This should log the same index as the log at https://github.com/veecle/tricore-probe/pull/37/files#diff-076d741b15a7f8c1976902c1b98e7ab83d29519ef4385c3ac6c758bbdf4b0a4eR262-R266

Reply:
On second thought, please remove that log line. Logging this on every iteration makes debug-logs very hard to read.